hazelcast/src/main/java/com/hazelcast/internal/util/UuidUtil.java

```diff
@@ -29,40 +27,45 @@
  * which are based on cryptographically weak {@link Random}
  * and strong {@link SecureRandom} pseudo random number generators.
  */
-@SuppressWarnings("unused")
 public final class UuidUtil {
 
+    /**
+     * The nil UUID is special form of UUID that is specified to have all
+     * 128 bits set to zero as described in the RFC 4122.
+     */
+    public static final UUID NIL_UUID = new UUID(0, 0);
+
     private UuidUtil() {
     }
 
     /**
-     * Creates a new cluster {@link UUID} string,
+     * Creates a new cluster {@link UUID},
      * based on a cryptographically weak pseudo random number generator.
      *
-     * @return a new cluster {@link UUID} string
+     * @return a new cluster {@link UUID}
      */
-    public static String createClusterUuid() {
-        return newUnsecureUuidString();
+    public static UUID createClusterUuid() {
```

Comment:
this and the following two methods are returning the same thing, can we merge them into single method, e.g. createUUID(). Or it can be removed totally and let the callers use newUnsecureUUID directly, there is no need for an extra redirection of methods.

Reply:
Okay, I will change that